src/sentry/integrations/jira/client.py

```diff
@@ -22,27 +22,23 @@ def md5(*bits):
     return _md5(':'.join((force_bytes(bit, errors='replace') for bit in bits)))
 
 
-class JiraApiClient(ApiClient):
-    COMMENT_URL = '/rest/api/2/issue/%s/comment'
-    STATUS_URL = '/rest/api/2/status'
-    CREATE_URL = '/rest/api/2/issue'
-    ISSUE_URL = '/rest/api/2/issue/%s'
-    META_URL = '/rest/api/2/issue/createmeta'
-    PRIORITIES_URL = '/rest/api/2/priority'
-    PROJECT_URL = '/rest/api/2/project'
-    SEARCH_URL = '/rest/api/2/search/'
-    VERSIONS_URL = '/rest/api/2/project/%s/versions'
-    USERS_URL = '/rest/api/2/user/assignable/search'
-    SERVER_INFO_URL = '/rest/api/2/serverInfo'
-    ASSIGN_URL = '/rest/api/2/issue/%s/assignee'
-    TRANSITION_URL = '/rest/api/2/issue/%s/transitions'
+class JiraCloud(object):
+    """
+    Contains the jira-cloud specifics that a JiraClient needs
+    in order to communicate with jira
+    """
 
-    def __init__(self, base_url, shared_secret):
-        self.base_url = base_url
+    def __init__(self, shared_secret):
         self.shared_secret = shared_secret
-        super(JiraApiClient, self).__init__(verify_ssl=True)
 
-    def request(self, method, path, data=None, params=None, **kwargs):
+    @property
+    def cache_prefix(self):
+        return 'sentry-jira-2:'
+
+    def request_hook(self, method, path, params, **kwargs):
```

Comment:
For whatever reason, when I read `request_hook` I thought hook as in webhook. It took me a second to realize what this was. I'm not sure if anyone else will have that bias though. Just thought I'd point it out.

Reply:
I am not super happy with this name either. If you have a suggestion I am happy to change it.